### PR TITLE
Updates code of conduct

### PR DESCRIPTION
**Pull request by brookr**

### CODE REVIEW

**Review by Rapparicio — APPROVED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -7,45 +7,51 @@ Anyone who violates this code of conduct may be immediately removed from any cur
...
+- **Be considerate**: Any decision you take will affect users and colleagues and you should take those consequences into account when making decisions. Remember that we’re a worldwide community, so you might not be communicating in someone else’s primary language. Your work will be used by other people and you in turn will depend on the work of others. 
+- **Be respectful**: Not all of us will agree all of the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It’s important to remember that a community where people feel uncomfortable or threatened is not a productive one. Your treatment of others shall not negatively impact other's ability to learn. Choose patience, and give others the benefit of the doubt.
```
Comment (johncokos) [inline README.md:16]: "is not a productive one" ...

This is a good opportunity to fold in the words "psychologically safe" - Maybe something like "is not a psychologically safe and therefore unproductive environment"

Just spitballing

Comment (johncokos) [reply README.md:16]: Also "give others the benefit of the doubt" might also include some language about space. Maybe something like 

"Choose patience, and provide others the space needed to think and speak, giving the benefit of the doubt whenever you can"

Comment (JB-Tellez) [reply README.md:16]: "**Be considerate**: Any decision you take will affect users and colleagues" 

replace "Any decision you take" with "Actions you take" or "Your words and actions will affect". Something to distinguish from "I made an internal decision to have Mexican vs. Thai for dinner tonight" ;)

File: README.md [hunk 2]
```diff
@@ -7,45 +7,51 @@ Anyone who violates this code of conduct may be immediately removed from any cur
...
+- **Give attribution, engage honestly**: Since the quality of our work often depends on the contributions of others, we strive to give proper attribution to all sources used, including chatbots, web articles, and Q&A sites (like Stack Overflow). As members of the community, we agree to represent ourselves truthfully, claim only work that is our own, properly attribute collaborations, and engage earnestly and honestly in all assignments.
```
Comment (JB-Tellez) [inline README.md:17]: Replace "claim only work that is our won" with "to only claim that is our own"

**Review by johncokos — CHANGES_REQUESTED** [review]

**Review by johncokos — APPROVED** [review]

**Pull request merged**